examples/vanilla-ts-esm/public/mux-player-renditions.html

```diff
@@ -46,7 +46,7 @@ <h1><a href="/">Elements</a></h1>
     <mux-player
       id="muxPlayer"
       stream-type="on-demand"
-      playback-id="23s11nz72DsoN657h4314PjKKjsF2JG33eBQQt6B95I"
+      playback-id="VcmKA6aqzIzlg3MayLJDnbF55kX00mds028Z65QxvBYaA"
```

Comment:
question(non-blocking): Is this a better playback-id to demo renditions or was this just for some testing? if the latter, consider reverting.

Reply:
I prefer this playback id for inspecting the quality selector UI. it doesn't have a black background.